.github/workflows/pub.yml

```diff
@@ -0,0 +1,63 @@
+name: Publish Release
+
+on:
+  push:
+    tags:
+    - 'v*'
+
+jobs:
+  pub_create_github_release:
+    name: Create Release
+    runs-on: ubuntu-latest
+    outputs:
+      upload_url: ${{ steps.create_release.outputs.upload_url }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: ${{ github.ref }}
+          draft: false
+          prerelease: false
+
+  pub_build:
+    name: Publish Build
+    runs-on: ubuntu-latest
+    needs: pub_create_github_release
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Install x86_64-pc-windows-gnu
+      run: rustup target add x86_64-pc-windows-gnu
+
+    - name: Build Release
+      run: |
+        chmod +x ./release.sh
```

Comment:
You can skip making this executable buy running it in the shell directly with `sh release.sh`. I'm actually surprised that this runs in a shell when executed.